src/display-buffer.coffee

```diff
@@ -841,9 +841,15 @@ class DisplayBuffer extends Model
     if screenLine.isSoftWrapped() and column >= maxScreenColumn
       if wrapAtSoftNewlines
         row++
-        column = 0
+        column = @screenLines[row].clipScreenColumn(0)
       else
         column = screenLine.clipScreenColumn(maxScreenColumn - 1)
+    else if screenLine.isColumnInsidePhantomToken(column)
+      if wrapAtSoftNewlines
```

Comment:
I would think this would be the other way around... when `wrapAtSoftNewlines` is `true` we're supposed to round *up* when clipping. Otherwise we round down. Am I missing something?